netman/adapters/switches/juniper/base.py

```diff
@@ -415,9 +415,7 @@ def edit_interface_spanning_tree(self, interface_id, edge=None):
 
     def set_interface_state(self, interface_id, state):
         update = Update()
-        update.add_interface(interface_main_update(interface_id, [
-            to_ele("<disable />") if state is OFF else to_ele("<enable />")
-        ]))
+        update.add_interface(interface_state_update(interface_id, state))
```

Comment:
Before it did :

    <interface>
        <name>Ge-0/0/99</name>
    </interface>
    <disable />

And now it does :

    <interface>
        <name>Ge-0/0/99</name>
        <disable />
    </interface>

?